src/controllers/collection.js

```diff
@@ -12,21 +12,28 @@ class CollectionController extends BaseController {
   }
 
   create (index, collection, mappings = {}, options = {}) {
-    return this.query({
+    const request = {
       index,
       collection,
       body: mappings,
-      action: 'create'
-    }, options)
+      action: 'create',
+      refresh: options.refresh
+    };
+    delete options.refresh;
```

Comment:
..why, since now Kuzzle.query handles the refresh option?

It seems to me that you don't need to change controllers to fix the issue.

Reply:
I changed the way it fix the issue in the last commit